mne/viz/_brain/_brain.py

```diff
@@ -1692,8 +1692,10 @@ def _clear_callbacks(self):
         # Remove the default key binding
         if getattr(self, "iren", None) is not None:
             try:
+                # pyvista<=0.30.0
                 self.plotter._key_press_event_callbacks.clear()
             except AttributeError:
+                # pyvista>=0.30.0
```

Comment:
`== 0.30.0` will match both though?? 😅

Reply:
Whoops

Reply:
It should be fixed in 6f98ca81ce618495e6bf1a3673cf637f64d0d45b

Reply:
should probably be `0.3.0` not `0.30.0`